backend/settings.py

```diff
@@ -13,7 +13,7 @@
 from pathlib import Path
 import environ
 
-env = environ.Env(
+ENV = environ.Env(
```

Comment:
I am not also very cumfortable with that, but I changed its name to `ENV` because with `env`, I wasn't able to access the variable from `api/config.py`.
But I think I just got why. Let me fix it.

Reply:
That's because settings name can't be lowercase if you want to import `settings` from  `django.conf`.

See https://docs.djangoproject.com/en/4.1/topics/settings/#creating-your-own-settings

Reply:
So you can create two variables `DATABASE_NAME = env("DATABASE_NAME")` and `DATABASE_URL = env("DATABASE_URL")`, and use `settings.DATABASE_NAME`